docs/release-notes/index.md

```diff
@@ -69,10 +69,8 @@ For product updates that are in progress or planned [visit the Codacy public roa
 ## Codacy Self-hosted release notes {: id="self-hosted"}
 
 v8
-<!--NOTE
-    Mention the change from https://github.com/codacy/chart/pull/744 in the release notes for the next Codacy Self-hosted version.
--->
 
+-   [v8.2.0](self-hosted/self-hosted-v8.0.0.md) (September 21, 2022)<!-- TODO Update release date -->
 -   [v8.1.0](self-hosted/self-hosted-v8.0.0.md) (June 17, 2022)
```

Comment:
links are pointing to v8.0.0

Reply:
Great catch, @pedrocodacy! :detective: It's a classic copy+paste error that was already starting to propagate to these release notes as well.